HPC/dev-guide.md

```diff
@@ -0,0 +1,51 @@
+## Compile instructions on an Ec2
+If you are a developer who want to build and test your applications on Graviton. You can get started by launching a Ubuntu 20.04 C7g instance from the console. Follow the procedures below to set up the tools:
+```
+# get the build tools and upgrade GCC
```

Comment:
Since these are code blocks and all github code blocks now have a copy-to-clipboard ability, can I easily take this shell snippet and copy it directly into my terminal to run?

Reply:
Yes.  You can copy the code blocks and run from an ubuntu 20.04 c7g.4xlarge; (4xlarge or larger machine is to speed up the compile and provide enough memory for some of the HPC unit tests).